randovania/games/samus_returns/layout/hint_configuration.py

```diff
@@ -12,10 +14,21 @@ class ItemHintMode(BitPackEnum, Enum):
     PRECISE = "precise"
 
     @classmethod
-    def default(cls) -> "ItemHintMode":
+    def default(cls) -> ItemHintMode:
+        return cls.PRECISE
+
+
+class BabyHintMode(BitPackEnum, Enum):
+    DISABLED = "disabled"
+    HIDE_AREA = "hide-area"
+    PRECISE = "precise"
```

Comment:
I don't see a reason why this needs to be a copy paste from `ItemHintMode`.

Reply:
I'd have to test without, but I was just going off of what prime 1 does. But might not be fully necessary